src/stories/components/CustomBarChart/CustomBarChart.tsx

```diff
@@ -32,12 +37,12 @@ export const CustomBarChart = (props: CustomBarChartProps) => {
     null
   );
   if (!props.items) return <span />;
-
+  const monthsProps = props?.months && props?.months.map((date) => date.toFormat('MMMM'));
```

Comment:
Please, note that this is equivalent to (in order to actually use optional change full potential):

```ts
const monthsProps = props?.months?.map((date) => date.toFormat('MMMM'));
```